charts/signoz/values.yaml

```
@@ -1210,29 +1210,55 @@ otelCollector:
   # -- Extra volumes for OtelCollector pod
   extraVolumes: []
   ingress:
-    # -- Enable ingress for OtelCollector
-    enabled: false
-    # -- Ingress Class Name to be used to identify ingress controllers
-    className: ""
-    # -- Annotations to OtelCollector Ingress
-    annotations: {}
-    # cert-manager.io/cluster-issuer: letsencrypt-prod
-    # nginx.ingress.kubernetes.io/ssl-redirect: "true"
-    # nginx.ingress.kubernetes.io/backend-protocol: "GRPC"
-    # kubernetes.io/ingress.class: nginx
-    # kubernetes.io/tls-acme: "true"
-    # -- OtelCollector Ingress Host names with their path details
-    hosts:
-      - host: otelcollector.domain.com
-        paths:
-          - path: /
-            pathType: ImplementationSpecific
-            port: 4318
-    # -- OtelCollector Ingress TLS
-    tls: []
-    #  - secretName: chart-example-tls
-    #    hosts:
-    #      - otelcollector.domain.com
+    grpc:
```

```suggestion
    grpc:
```

Hey @suyash-811,
Let's keep the original intact for GRPC and add HTTP. Removing the older ingress will break compatibility for a lot of users.

---

understood. will add a new section for http then